src/i18n/nb_NO.ts

```diff
@@ -369,237 +247,115 @@ export default {
-            credits: 20_000,
-            minPersonnel: 0,
-            maxPersonnel: 0,
-            schooling: 'Fire Station - Heavy Machinery Operating',
-            shownSchooling: 'Heavy Machinery Operating',
-            special:
-                "Dozer Trailer that can be towed with the Crew cap semi. Requires Heavy Machinery Operating and Truck Driver's License trained personnel to operate.<br> The dozer trailer cannot be assigned personnel, <br>either assign your trained personnel to the towing vehicle, or assign them to a separate unit en route.",
-        },
-        41: {
-            caption: 'Crew cap semi',
-            color: '#0018b8',
-            coins: 12,
-            credits: 5_000,
-            minPersonnel: 1,
-            maxPersonnel: 3,
-            schooling: "Fire Station - Truck Driver's License",
-            shownSchooling: "Truck Driver's License",
-        },
-        42: {
-            caption: 'FBI Unit',
-            color: '#001bcc',
-            coins: 15,
-            credits: 10_000,
-            minPersonnel: 2,
-            maxPersonnel: 4,
-        },
-        43: {
-            caption: 'FBI Investigation Wagon',
-            color: '#001ee0',
-            coins: 15,
-            credits: 10_000,
-            minPersonnel: 2,
-            maxPersonnel: 4,
-        },
-        44: {
-            caption: 'FBI Mobile Command Center',
-            color: '#0021f5',
-            coins: 25,
-            credits: 25_000,
-            minPersonnel: 1,
-            maxPersonnel: 4,
-            schooling: 'Police - FBI Mobile Center Commmander',
-            shownSchooling: 'FBI Mobile Center Commmander',
-        },
-        45: {
-            caption: 'FBI Bomb Technician Vehicle',
-            color: '#0a2bff',
-            coins: 35,
-            credits: 35_500,
             minPersonnel: 1,
             maxPersonnel: 2,
-            schooling: 'Police - FBI Bomb Technician',
-            shownSchooling: 'FBI Bomb Technician',
+            schooling: 'Politi - Utrykningspoliti',
+            shownSchooling: 'Utrykningspoliti',
         },
-        46: {
-            caption: 'FBI Surveillance Drone',
-            color: '#f04242',
-            coins: 25,
-            credits: 25_000,
-            minPersonnel: 1,
-            maxPersonnel: 1,
-            wtank: 0,
-            schooling: 'Police - FBI Drone Operator',
-            shownSchooling: 'FBI Drone Operator',
-        },
-        47: {
-            caption: 'Sheriff Unit',
+        22: {
+            caption: 'Utrykningsenhetbil',
             color: '#ad0e0e',
-            coins: 20,
-            credits: 15_000,
+            coins: 15,
+            credits: 19_000,
             minPersonnel: 1,
             maxPersonnel: 2,
-            schooling: 'Police - Sheriff',
-            shownSchooling: 'Sheriff',
+            schooling: 'Politi - Våpentrening',
+            shownSchooling: 'Våpentrening',
         },
     },
     buildings: {
```

Comment:
the building order is wrong. look here: view-source:https://www.nodsentralspillet.com/buildings/new . after the id's you have to go

Reply:
That is not resolved. look at the picture the values these are the building id's

Reply:
I don't understand what you mean here, your example doesn't give me any information other than the list inside ny bygning. https://i.imgur.com/sAf1zJs.png
And that list is corresponding with the list in nb_NO.ts

Reply:
so here all id's
0 = Brannstasjon
1 = Brannskole
2 = Ambulansestasjon
4 = Sykehus
5 = Helikopterstasjon
6 = Politistasjon
7 = Nødetatssenter
8 = Politiskole
13 = Politiheliport
14 = Oppstillingsplass
16 = Fengsel
18 = Brannstasjon (liten)
19 = Politistasjon (liten)
20 = Ambulansestasjon (liten)

Reply:
Fixed, I feel stupid now - only saw one of your pictures 🤦